infra/resources/_modules/apim/api/ioweb/user-function/_swagger.json

```diff
@@ -0,0 +1,97 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "IO Wallet - User API",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/wallet-instances/current/status": {
+      "get": {
+        "summary": "Retrieve the current Wallet Instance status",
+        "operationId": "getCurrentWalletInstanceStatus",
+        "security": [
+          {
+            "FunctionsKey": [],
```

Comment:
should we remove `FunctionsKey`?

Reply:
Actually we have used the same swagger configuration of the user_func, but if is not necessary we can remove the entire `security` block for both

Reply:
but unlike the function, in the APIM it's not necessary to use the` x-functions-key` header for authentication, right?

Reply:
@silvicir it depends on the `subscription_required` flag set at API group level. In this PR, it is disabled and then there is no authentication